src/modules/Pagination/Pagination.scss

```diff
@@ -0,0 +1,63 @@
+@import 'index.scss';
+
+.j-pagination {
+  .Pagimagic-nav {
+    &-item {
+    	display: inline-block;
+      vertical-align: top;
+      height: 30px;
+      line-height: 30px;
+      background-color: $hf-color-white;
+	    border: $hf-border;
+	    font-size: 12px;
+	    text-align: center;
+  	  color: $hf-color-gray-medium;
+      padding: 0 8px;
+      box-sizing: border-box;
+      transition: background-color 0.05s, color 0.05s, border-color 0.05s;
+      cursor: pointer;
+
+      &.active {
+        background-color: $hellofresh-color;
+        border-color: $hellofresh-color;
+        color: $hf-color-white;
+      }
+
+      &--prev,
+      &--next {
+        position: relative;
+        width: 20px;
+        height: 30px;
+        border: none;
+        background: none;
+
+        &:before {
+          content: '';
+          display: block;
+          position: absolute;
+          width: 8px;
```

Comment:
is this all for centering the arrow? maybe using flex on a normal element it'll be easier, in case not seems like all values depend on each other, maybe have a mixing or something to generate this worst case scenario.